aws/vars.tf

```diff
@@ -156,11 +122,75 @@ variable "prometheus_instance_type" {
   default     = "c5.2xlarge"
 }
 
+variable "cluster_ami" {
+  description = "AMI for Redpanda broker nodes (if not set, will select based on the client_distro variable"
+  default     = null
+}
+
+variable "prometheus_ami" {
+  description = "AMI for prometheus nodes (if not set, will select based on the client_distro variable"
+  default     = null
+}
+
+variable "client_ami" {
+  description = "AMI for Redpanda client nodes (if not set, will select based on the client_distro variable"
+  default     = null
+}
+
 variable "public_key_path" {
   description = "The public key used to ssh to the hosts"
   default     = "~/.ssh/id_rsa.pub"
 }
 
+data "aws_ami" "ami" {
+    most_recent = true
+
+    filter {
+        name   = "name"
+        values = ["ubuntu/images/hvm-ssd/ubuntu-*-amd64-server-*", "Fedora-Cloud-Base-*.x86_64-hvm-us-west-2-gp2-0", "debian-*-amd64-*", "debian-*-hvm-x86_64-gp2-*'", "amzn2-ami-hvm-2.0.*.0-x86_64-gp2", "RHEL*HVM-*-x86_64*Hourly2-GP2"]
```

Comment:
`amzn2-ami-hvm-2.0.*.0-x86_64-gp2` should be  `amzn2-ami-hvm-2.0.*-x86_64-gp2` (note the removed .0) otherwise it appears to miss potentially the latest version of a given weekly ami release.

Reply:
Done